backend/src/test/java/botobo/core/acceptance/DomainAcceptanceTest.java

```diff
@@ -96,6 +100,59 @@ protected User anyUser() {
                 .build();
     }
 
+    public CardResponse 카드_등록되어_있음(String question, String answer, Long workbookId, String accessToken) {
+        CardRequest cardRequest = CardRequest.builder()
+                .question(question)
+                .answer(answer)
+                .workbookId(workbookId)
+                .build();
+        return 카드_등록되어있음(cardRequest, accessToken);
+    }
+
+    private CardResponse 카드_등록되어있음(CardRequest cardRequest, String accessToken) {
+        return 카드_생성_요청(cardRequest, accessToken).convertBody(CardResponse.class);
+    }
+
+
+    private RequestBuilder.HttpResponse 카드_생성_요청(CardRequest cardRequest, String accessToken) {
+        return request()
+                .post("/api/cards", cardRequest)
+                .auth(accessToken)
+                .build();
+    }
+
+    protected WorkbookResponse 유저_문제집_등록되어_있음(String name, boolean opened, List<TagRequest> tags, String accessToken) {
+        WorkbookRequest workbookRequest = WorkbookRequest.builder()
+                .name(name)
+                .opened(opened)
+                .tags(tags)
+                .build();
+        return 유저_문제집_등록되어_있음(workbookRequest, accessToken);
+    }
+
+    protected WorkbookResponse 유저_문제집_등록되어_있음(WorkbookRequest workbookRequest, String accessToken) {
+        return 유저_문제집_생성_요청(workbookRequest, accessToken).convertBody(WorkbookResponse.class);
+    }
+
+    protected RequestBuilder.HttpResponse 유저_문제집_생성_요청(WorkbookRequest workbookRequest, String accessToken) {
+        return request()
+                .post("/api/workbooks", workbookRequest)
+                .auth(accessToken)
+                .build();
+    }
+
+    protected WorkbookResponse 유저_문제집_등록되어_있음(String name, boolean opened, String accessToken) {
+        List<TagRequest> tagRequests = Collections.singletonList(
+                TagRequest.builder().id(1L).name("자바").build()
+        );
+        WorkbookRequest workbookRequest = WorkbookRequest.builder()
+                .name(name)
+                .opened(opened)
+                .tags(tagRequests)
+                .build();
+        return 유저_문제집_등록되어_있음(workbookRequest, accessToken);
+    }
```

Comment:
```suggestion
    protected WorkbookResponse 유저_문제집_등록되어_있음(String name, boolean opened, String accessToken) {
        List<TagRequest> tagRequests = Collections.singletonList(
                TagRequest.builder().id(1L).name("자바").build()
        );
        return 유저_문제집_등록되어_있음(name, opened, tagRequests, accessToken);
    }
```
만들어놓은 메서드를 재활용 해보는 건 어떨까요? 😁
그리고 이건 개인적인 생각인데, 제가 이 메서드를 쓰게 된다면 파라미터로 TagRequest 를 받지 않으니까
문제집을 만들 때 태그를 안넣을 것 같다는 예상을 할 것 같아요. 이름을 바꿔도 괜찮을 거 같은데, 오즈가 편한대로 하면됩니다!

Reply:
아마 중간곰이 name이랑 opened만 넣어도 쉽게 문제집 등록되도록 만든거지 않나 싶슴다
괜찮겠다 싶어서 그냥 뒀는데 피드백 반영해서 메서드 이름을 변경해줬습니다ㅋㅋㅋ